app/services/transfers/npq_participants.rb

```diff
@@ -2,31 +2,36 @@
 
 module Transfers
   class NPQParticipants
```

Comment:
could we avoid changes to this service as it's a tool we use and not hooked up to anything?